Apply COPYRIGHT header to files

philipcristiano opened the pull request:
Automated using https://github.com/osterman/copyright-header

```
copyright-header --license-file LICENSE.short \
                 --copyright-holder 'Reactive Ops Inc.' \
                 --copyright-software 'Pentagon' \
                 --copyright-year 2017 \
                 --add-path bin:build:config:deploy:dist:docker:docs:examples:lib \
                 --output-dir ../
```

LICENSE.short contains

```
Copyright 2017 Reactive Ops Inc.

Licensed under the Apache License, Version 2.0 (the “License”);
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an “AS IS” BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.Licensed under the Apache License, Version 2.0 (the “License”);
```

Files were skipped and the LICENSE will need to be added manually

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/reactiveops/pentagon?pullRequest=1) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/reactiveops/pentagon?pullRequest=1) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/reactiveops/pentagon?pullRequest=1) it.</sub>

philipcristiano committed:
Apply COPYRIGHT header to files

Automated using https://github.com/osterman/copyright-header

```
copyright-header --license-file LICENSE.short \
                 --copyright-holder 'Reactive Ops Inc.' \
                 --copyright-software 'Pentagon' \
                 --copyright-year 2017 \
                 --add-path bin:build:config:deploy:dist:docker:docs:examples:lib \
                 --output-dir ../
```

LICENSE.short contains

```
Copyright 2017 Reactive Ops Inc.

Licensed under the Apache License, Version 2.0 (the “License”);
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an “AS IS” BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.Licensed under the Apache License, Version 2.0 (the “License”);
```

Files were skipped and the LICENSE will need to be added manually
